gateway-js/src/index.ts

```diff
@@ -351,24 +430,41 @@ export class ApolloGateway implements GraphQLService {
       }
     }
 
-    await this.updateComposition();
-    if (
-      (isManagedConfig(this.config) || this.experimental_pollInterval) &&
-      !this.pollingTimer
-    ) {
-      this.pollServices();
-    }
+    const result = isStaticConfig(this.config)
+      ? this.loadStatic()
+      : await this.loadDynamic();
 
     const mode = isManagedConfig(this.config) ? 'managed' : 'unmanaged';
-
     this.logger.info(
       `Gateway successfully loaded schema.\n\t* Mode: ${mode}${
-        (this.apolloConfig && this.apolloConfig.graphId)
+        this.apolloConfig && this.apolloConfig.graphId
           ? `\n\t* Service: ${this.apolloConfig.graphId}@${this.apolloConfig.graphVariant}`
           : ''
       }`,
     );
 
+    return result;
+  }
+
+  private loadStatic() {
```

Comment:
Maybe worth calling out with a comment that we're not doing anything because this already got initialized in the constructor?

Reply:
No longer the case via 525df37